text/0107-construct-library-module-lifecycle.md

```diff
@@ -30,20 +30,27 @@ module:
 - Stage 2 - Developer Preview
 - Stage 3 - GA
 
+Each stage in this document describes 3 distinct sets of activities:
+
+* **Entry activities:** Actions that need to be taken before the module enters this stage. These can either be automatic, or done by the team. They will usually be performed once a module is deemed for stage maturation, and all its exit criteria are met.
```

Comment:
"They will usually be performed once a module is deemed for stage maturation"

"They will be performed as soon as the module moves into the new stage"

Reply:
👍